torchaudio/sox_effects.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import division, print_function
+import torch
+import _torch_sox
+
+import torchaudio
+
+
+def effect_names():
+    """Gets list of valid sox effect names
+
+    Returns: list[str]
+
+    Example::
+        >>> EFFECT_NAMES = torchaudio.sox_effects.effect_names()
+    """
+    return _torch_sox.get_effect_names()
+
+
+def SoxEffect():
+    """Create an object for passing sox effect information between python and c++
+
+    Returns: SoxEffect(object)
+      - ename (str), name of effect
+      - eopts (list[str]), list of effect options
+    """
+    return _torch_sox.SoxEffect()
+
+
+class SoxEffectsChain(object):
+    """SoX effects chain class.
```

Comment:
An example in the documentation would be nice. Also probably mention at this point that initialize_sox() needs to be called before, if I understand correctly.

Reply:
That's a good idea.  I'll add that.